### PR TITLE
development/gtest: Rework libraries installation.

### PR DESCRIPTION
**Pull request by Ponce**
Copy also missing cmake files

### CODE REVIEW

**Comment by willysr** [comment]
@sbo-bot: build development/gtest

File: development/gtest/gtest.SlackBuild [hunk 1]
```diff
@@ -111,13 +111,20 @@ install -m 0644 googletest/include/$PRGNAM/internal/custom/*.h \
...
+  ln -s $(basename $i .so).so.$VERSION $PKG/usr/lib$LIBDIRSUFFIX/$(basename $i .so).so
+done
 mkdir -p $PKG/usr/lib$LIBDIRSUFFIX/pkgconfig
 for i in gmock gmock_main gtest gtest_main ; do
   sed "s| -lpthread||g" build/googletest/generated/${i}.pc \
     > $PKG/usr/lib$LIBDIRSUFFIX/pkgconfig/${i}.pc
 done
 
+# copy also the cmake files
+cp -a build/googletest/CMakeFiles/Export/lib64/cmake $PKG/usr/lib$LIBDIRSUFFIX/
```
Comment (willysr) [inline development/gtest/gtest.SlackBuild:125]: you put hard-coded lib64 there @Ponce

Comment (Ponce) [reply development/gtest/gtest.SlackBuild:125]: ops, I'll do a new pull request

Comment (willysr) [reply development/gtest/gtest.SlackBuild:125]: you can just simply push -f

Comment (Ponce) [reply development/gtest/gtest.SlackBuild:125]: ok, will do next time

**Pull request closed without merging**